src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ChangeEvent } from "react";
+import { SliderContainer, BunnyButt, BunnySlider, BarBackground, BarProgress, StyledInput, SliderLabel } from "./styles";
+import SliderProps from "./types";
+
+// Using require instead of import to avoid trouble with TS bunding and SVG types
+/* eslint-disable @typescript-eslint/no-var-requires */
+const bunnyButt = require("./svg/bunnybutt.svg");
```

Comment:
The reason for doing it like this is because I need the link to the svg in the css, I'm not using the svg as a react component. If you got a preferred solution for assets I could change it, but I don't thing react-svg will work :/

Reply:
as far as I see, you are just using this as a `src` props for an image component, which is exactly the same as using the Svg components

Reply:
you're right! Fixing this